lib/galaxy/tools/parameters/wrapped_json.py

```diff
@@ -79,20 +79,25 @@ def _data_input_to_path(v):
         if handle_files == "skip":
             return SKIP_INPUT
         raise NotImplementedError()
-    elif input_type in ["select", "text", "color", "hidden"]:
-        json_value = _cast_if_not_none(value, str)
+    elif input_type in ["text", "color", "hidden"]:
+        json_value = _cast_if_not_none(value_wrapper, str)
     elif input_type == "float":
-        json_value = _cast_if_not_none(value, float, empty_to_none=True)
+        json_value = _cast_if_not_none(value_wrapper, float, empty_to_none=True)
     elif input_type == "integer":
-        json_value = _cast_if_not_none(value, int, empty_to_none=True)
+        json_value = _cast_if_not_none(value_wrapper, int, empty_to_none=True)
     elif input_type == "boolean":
-        json_value = _cast_if_not_none(value, bool)
+        json_value = _cast_if_not_none(value_wrapper, bool)
+    elif input_type == "select":
+        if input.multiple:
+            json_value = [_ for _ in _cast_if_not_none(value_wrapper.value, list)]
```

Comment:
If you can get the tests working, this change definitely makes sense to me!

Reply:
Guess this is possible. 

I just would like to use the opportunity to remark that this PR (in addition to fix the "None" problem for selects) also changes the following:

- multiple selects are now lists (before comma separated string)

Also I'm not sure if data_column should be cast to int, because of `<param type="data_column" 
use_header_names="true".../>` (but I could not get this attribute working as expected anyway).